pkgs/applications/science/machine-learning/streamlit/default.nix

```
@@ -0,0 +1,34 @@
+{   lib, buildPythonApplication, fetchPypi
+  , altair, astor, base58, blinker, boto3, botocore, click, enum-compat
+  , future, pillow, protobuf, requests, toml, tornado, tzlocal, validators, watchdog
+  , jinja2, setuptools
+}:
+
+buildPythonApplication rec {
+  pname = "streamlit";
+  version = "0.49.0";
+  format = "wheel";
```

do you mind leaving a comment as to why wheel is chosen? it's highly not preferred, but seems to be the only distribution
```suggestion
  format = "wheel"; # only distribution available
```